src/components/TrialBanner.js

```diff
@@ -156,6 +157,11 @@ export const TrialBanner = _.flow(
           button.isExternal ? icon('pop-out', { style: { marginLeft: '0.25rem' } }) : null
         ]),
         div({ style: { marginLeft: '3rem', display: 'flex', flexDirection: 'column', alignItems: 'flex-end' } }, [
+          (trialState === 'Enrolled') && h(Clickable, {
+            style: { borderBottom: 'none' },
+            tooltip: 'Hide banner',
+            onClick: () => this.setState({ snoozeBanner: true })
+          }, [icon('times-circle', { size: 25, style: { fontSize: '1.5rem', cursor: 'pointer', strokeWidth: 1.5 } })]),
```

Comment:
Nit: I don't think `strokeWidth` is doing anything to the icon (which is unfortunate). Should remove it for clarity.